src/main/java/com/corpo/coliseum/domain/service/impl/UserServiceImpl.java

```diff
@@ -16,15 +17,18 @@
     @Inject
     ModelMapper modelMapper;
 
+    private static final Logger LOG = Logger.getLogger(UserServiceImpl.class);
+
     @Override
     @Transactional
     public void signUp(String mail, String name) throws UserException {
-        if(!userAlreadySignIn(mail)){
+        if(Boolean.FALSE.equals(userAlreadySignIn(mail))){
             User.builder()
```

Comment:
Same as above

Reply:
more easily to read, avoid skipping ! if you read this fast